lib/matplotlib/projections/geo.py

```diff
@@ -190,7 +190,7 @@ def set_longitude_grid(self, degrees):
         """
         Set the number of degrees between each longitude grid.
         """
-        number = (360.0 / degrees) + 1
+        number = int(360.0 / degrees) + 1
```

Comment:
I don't understand; if degrees = 360, it's a nice round division, so it doesn't matter if you use `ceil`, `round`, or plain `int`.